src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -557,13 +556,26 @@ def render_worker_node_specific(config, args):
                               "./deploy/cloud-config/cloud_init_worker_{}.txt".format(sku), config)
 
 
+def render_elasticsearch_node_specific(config, args):
+    if not "elasticsearch_node" in config or len(config["elasticsearch_node"]) == 0:
+        print("Warning: no elasticsearch node specified, logging service might not work as expected.")
+        return
+    config.pop("mount_and_link", [])
+    config = escaped_etcd_end_point_and_k8s_api_server(config)
+    example_node_name = config["elasticsearch_node"][0].split(".")[0]
+    common_labels = get_kube_labels_of_machine_name(config, example_node_name)
+    default_f2cp = ["kubernetes_common", "kubelet_worker"]
+    config["kube_labels"] = common_labels
+    config["file_modules_2_copy"] = [mod for mod in default_f2cp]
```

Comment:
` config["kube_labels"] = get_kube_labels_of_machine_name(config, example_node_name)`
`config["file_modules_2_copy"] = ["kubernetes_common", "kubelet_worker"]`?

Reply:
[mod for mod in default_f2cp] is a deepcopy, otherwise later the list would be overwriten, so this is used to avoid unexpected influence on other rendered files.